cclient/request/add_neighbors.h

```diff
@@ -13,7 +13,7 @@ typedef struct {
   /**
    * List of neighbors URIs
    */
-  string_array uris;
+  tryte_array uris;
```

Comment:
I know that you just changed this, but ... taking everything a step further, the req/res types should not know/care about trytes. they should only use flextrits?